client/src/pages/Testing/RunTest.tsx

```diff
@@ -144,15 +172,18 @@ function UploadZipFile({
   );
 }
 
-function UploadFileWarning() {
+type UploadFileProps = {
+  envIsLocal: boolean;
+};
+function UploadFileWarning({ envIsLocal }: UploadFileProps) {
   return (
     <div className="bg-state-error-lighter rounded p-4">
       <p className="text-state-error-dark flex flex-col gap-3">
         <span className="flex items-center gap-2">
           <WarningIcon aria-label="Warning" />
           <span>This environment is not approved to handle PHI/PII.</span>
         </span>
-        <span className="font-bold">
+        <span className={`${envIsLocal ? ' ' : 'text-center'} font-bold`}>
```

Comment:
The non-centered text on this prompt looked weird so I added this conditional style